presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -112,6 +113,13 @@ public void testForeignTable()
         execute("DROP SERVER devnull");
     }
 
+    @Test
+    public void testSystemTable()
+            throws Exception
+    {
+        assertGreaterThan(computeActual("SHOW TABLES FROM pg_catalog").getRowCount(), 0);
```

Comment:
- verify that `SHOW TABLES FROM pg_catalog` returned some well-known names (just 2-3 examples, not all)
- add a test for reading from such a table `SELECT * FROM pg_catalog.xxxx`